python/requirements/orca/requirement_extra.txt

```diff
@@ -0,0 +1,14 @@
+bert
+boto3
+cloudpickle
+fsspec
+onnx
+onnxruntime
+Pillow
+py4j
+pyarrow
+tensorflow
+tensorflow_estimator
```

Comment:
will tensorflow_estimator be installed as a dependency of tensorflow?

Reply:
I think so, do we need to remove all the dependencies of tensorflow and pytorch?

Reply:
In this file, any other dependencies of tensorflow and pytorch?

Reply:
No, but in requirement_base.txt， `packaging`, `numpy`, `six` are also in the requirements of tensorflow.

Reply:
Then it doesn't matter. Even if we don't install tensorflow, we have to install numpy and six.